test/ResponseTest.php

```diff
@@ -176,6 +177,60 @@ public function testChunkedResponseCaseInsensitiveZF5438()
         $this->assertEquals('c0cc9d44790fa2a58078059bab1902a9', md5($res->getContent()));
     }
 
+    /**
+     * @param int $chunksize the data size of the chunk to create
+     * @return string a chunk of data for embedding inside a chunked response
+     */
+    private function makeChunk($chunksize)
+    {
+        $chunkdata = str_repeat('W', $chunksize);
+        return sprintf("%d\r\n%s\r\n", $chunksize, $chunkdata);
+    }
+
+    /**
+     * @param Response $response
+     * @return float the time that calling the getBody function took on the response
+     */
+    private function getTimeForGetBody(Response $response)
+    {
+        $timeStart = microtime(true);
+        $response->getBody();
+        return microtime(true) - $timeStart;
+    }
+
+    /**
+     * @small
```

Comment:
Yes, without the path the ratios are on my machine 2000-5000 under PHP7 and about 250 on PHP5.6 ... Test runs > 60 secs.  

I can increase the ratio here to make the test pass, but there must be some other problem. I would not expect a ratio much larger than 1, after all this is only one iteration more than before.